### PR TITLE
change some rosacli jobs name and add the rosacli test workflow

### PR DESCRIPTION
**Pull request by yuwang-RH**
change some rosacli jobs name and add the rosacli test workflow

### CODE REVIEW

**Comment by yuwang-RH** [comment]
@yasun1 @xueli181114  Could you help review this PR?

**Comment by yuwang-RH** [comment]
@kuiwang02 @liangxia Could you help review and approve this PR? Thanks.

**Comment by kuiwang02** [comment]
I am ok from golang's view.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml [hunk 1]
```diff
@@ -31,37 +31,45 @@ tests:
     test:
     - ref: openshift-extended-rosacli-tests
     workflow: rosa-aws-sts-rosacli-e2e
-- as: aws-rosa-non-sts-advanced-f3
+- as: aws-rosa-non-sts-advanced-rosacli-e2e-f3
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml:34]:
```
-  name: periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-advanced-f3
+  name: periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-advanced-rosacli-e2e-f3
```
The job name already contains `rosacli` ( comes from the config file name). Perhaps no need to duplicate that ?

Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml:34]: We could have multiple job configures for rosacli-e2e and ocp-e2e, `aws-rosa-sts-advanced-rosacli-e2e` should easily for us to separate from the ocp one `aws-rosa-sts-advanced` .

Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml:34]: Just rethinking, agree with you. Let us remove `rosacli-e2e` from the job's name.

**Comment by yuwang-RH** [comment]
@yasun1 Could you help review? I updated the jobs name without -rosacli-

**Pull request merged**